lmp/static_deltas.py

```diff
@@ -75,6 +76,31 @@ def upload_delta_stats(factory: str, delta_stats: dict, tok_secret_name: str):
                             f"expected: {len(s['canonical-json'])}, received: {r.json()['size']}")
 
 
+def add_delta_stat_refs_to_targets(creds_zip_file: str, delta_stats: dict,
+                                   tuf_repo: str = "./tuf-repo"):
+    cmd("garage-sign", "init", "--repo", tuf_repo, "--credentials", creds_zip_file)
+    cmd("garage-sign", "targets", "pull", "--repo", tuf_repo)
+    targets_file = f"./tuf/{tuf_repo}/roles/unsigned/targets.json"
+    with open(targets_file) as f:
+        targets = json.load(f)
+
+    for target_name, target_value in targets["targets"].items():
+        to_hash = target_value["hashes"]["sha256"]
+        if to_hash in delta_stats:
+            target_value["custom"]["delta-stats"] = {
+                "sha256": delta_stats[to_hash]["sha256"],
+                "size": len(delta_stats[to_hash]["canonical-json"])
+            }
```

Comment:
so we set the sha256 and size and aklite will know how to download this as content addressable and do a limited read of "size" to pull down this delta info?

Reply:
Exactly, https://github.com/foundriesio/aktualizr-lite/blob/d13145f9b3b65897f3668ca0c2255c042b510904/src/rootfstreemanager.cc#L267